frontend/js/src/cb-review/CBReviewModal.tsx

```diff
@@ -514,7 +514,7 @@ export default NiceModal.create(({ listen }: CBReviewModalProps) => {
         </div>
 
         <div className="checkbox">
-          <label>
+          <label htmlFor="#acceptLicense">
```

Comment:
The ne testing framework has a strong emphasis on testing for accessibility as we write tests (pushed for targeting elements using accessibility features such as input labels, etc.)
Correcting the label here which was missing the `for` attribute targeting the actual checkbox input.